queue/queue.go

```diff
@@ -53,7 +56,7 @@ type inMemoryQueueItem struct {
 
 // New creates a new queue with a Storage specified in argument
 // A standard InMemoryQueueStorage is used if Storage argument is nil.
-func New(threads int, s Storage) (*Queue, error) {
+func New(threads int, randomDelay time.Duration, s Storage) (*Queue, error) {
```

Comment:
`Queue.RandomDelay` is a public member of `Queue`, it isn't necessary to initialize `RandomDelay` here. Please remove the `randomDelay` argument to avoid API change.